docs/en/observability/aggregation-options.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[aggregation-options]]
+= Aggregation options
+
+Aggregations summarize your data to make it easier to analyze.
+In some alerting rules, you can specify aggregations to gather data for the rule.
+
+The following aggregations are available in some rules:
+
+|===
+|Aggregation | Description
```

Comment:
I think the current structure works OK as long as we provide a summary table like this. I don't know if this table is complete, though. @jasonrhodes @maryam-saeidi Can you help me flesh this table out if anything is missing?

Reply:
I like the new [structure](https://observability-docs_bk_3676.docs-preview.app.elstc.co/guide/en/observability/master/aggregation-options.html) 👍🏻 

Just an idea:

|Aggregation|Current|Suggestion|
|---|---|---|
|Document count|Total number of documents in a field.|Number of documents in your dataset|

Reply:
Or "in the selected dataset"

Reply:
Agree, same for sum: 
> Total of a numeric field in ~~your~~ selected dataset.